### ISSUE TITLE
Sofia_Sans_Italic_MM.glyphs: fontmake build error

### ISSUE THREAD

**Issue by eliheuer** [lettersoup/Sofia-Sans#4]
When building `Sofia_Sans_Italic_MM.glyphs` with [fontmake](https://github.com/googlefonts/fontmake), I get the following error: 
```
Traceback (most recent call last):
  File "/Users/Air/Py/Venvs/type/bin/fontmake", line 10, in <module>
    sys.exit(main())
  File "/Users/Air/Py/Venvs/type/lib/python3.7/site-packages/fontmake/__main__.py", line 391, in main
    project.run_from_glyphs(glyphs_path, **args)
  File "/Users/Air/Py/Venvs/type/lib/python3.7/site-packages/fontmake/font_project.py", line 764, in run_from_glyphs
    self.run_from_designspace(designspace_path, **kwargs)
  File "/Users/Air/Py/Venvs/type/lib/python3.7/site-packages/fontmake/font_project.py", line 914, in run_from_designspace
    **kwargs
  File "/Users/Air/Py/Venvs/type/lib/python3.7/site-packages/fontmake/font_project.py", line 969, in _run_from_designspace_interpolatable
    designspace, output_path=output_path, output_dir=output_dir, **kwargs
  File "/Users/Air/Py/Venvs/type/lib/python3.7/site-packages/fontmake/font_project.py", line 411, in build_variable_font
    inplace=True,
  File "/Users/Air/Py/Venvs/type/lib/python3.7/site-packages/ufo2ft/__init__.py", line 530, in compileVariableTTF
    baseUfo = getDefaultMasterFont(designSpaceDoc)
  File "/Users/Air/Py/Venvs/type/lib/python3.7/site-packages/ufo2ft/util.py", line 384, in getDefaultMasterFont
    "Can't find base (neutral) master in DesignSpace document"
ufo2ft.errors.InvalidDesignSpaceData: Can't find base (neutral) master in DesignSpace document
```
I'm looking into it and will make a pull request with a fix if I find one.

**Comment by Eigi** [comment]
This should be fixed with pull request #13

**Comment by eliheuer** [comment]
Thanks! I remember this font had some designspace issues that blocked progress. I'll take a look at this new pull request today.
